packages/networks/ethereum/index.ts

```diff
@@ -20,5 +20,13 @@ export const ethereumNetwork: NetworkInfo = {
   theGraphEndpoint:
     "https://api.studio.thegraph.com/query/40379/teritori-mainnet/v1",
   vaultContractAddress: "0x6251B3384c8eD53e2Cc38d34c1f26ffE8d461B94",
-  riotContractAddress: "0x8f8304ea566affeb96ad0ffb593bbebd8876d124",
+  riotContractAddressGen0: "0x8f8304ea566affeb96ad0ffb593bbebd8876d124",
+  riotSquadStakingContractAddress: "<must_fill>",
+  // Substreams
+  firehoseEndpoint: "<must_fill>",
```

Comment:
these will come from where?

Reply:
I will fill this once we test everything ok and once we have the contract mainnet ETH deployed